pokemongo_bot/__init__.py

```diff
@@ -34,7 +34,7 @@ def work_on_cell(self, cell, position, include_fort_on_path):
         if (self.config.mode == "all" or self.config.mode == "poke") and 'catchable_pokemons' in cell:
             print '[#] Something rustles nearby!'
             for pokemon in cell['catchable_pokemons']:
-                with open('web/catchable.json', 'w') as outfile:
+                with open('web/info.json', 'w') as outfile:
```

Comment:
@earthchie I'm afraid you're going to break map functionality here. Remove this change from the merge request?